foundations/introduction/asking_for_help.md

```diff
@@ -57,4 +57,5 @@ The following questions are an opportunity to reflect on key topics in this less
 
 This section contains helpful links to related content. It isn't required, so consider it supplemental.
 
-- It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.
+- An encouraging article from Stanford on [Asking for help is hard — but people want to help more than we realize](https://news.stanford.edu/stories/2022/09/asking-help-hard-people-want-help-realize).
+- [The Mental Battle behind Asking for Help](https://hbr.org/2023/07/why-its-so-hard-to-ask-for-help) — Harvard Business Review article exploring the psychological barriers to asking for assistance.
```

Comment:
The wording needs to be updated here. I don't like the format that you are using here. Can you update this a complete sentence, something like this:
```suggestion
- Harvard Business Review article exploring [The Mental Battle behind Asking for Help](https://hbr.org/2023/07/why-its-so-hard-to-ask-for-help).